.github/workflows/tests.yml

```diff
@@ -28,20 +52,70 @@ jobs:
           key: ${{ matrix.job.os }}-m2-${{ hashFiles('**/pom.xml') }}
           restore-keys: ${{ matrix.job.os }}-m2
 
-      - name: Compile and install everything
-        run: mvn clean install -DskipTests
-
       - name: Run Andy unit tests
         run: mvn test -pl andy -DexcludedGroups=assignments
 
+  run_andy_tests:
```

Comment:
Can we call this assignment tests? When I read Andy tests, I think of Andy's unit tests.